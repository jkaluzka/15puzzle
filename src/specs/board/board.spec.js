import React from 'react';
import Board from '../../views/board/board';
import renderer from 'react-test-renderer';

describe('Board', () => {
  it('renders Board component correctly', () => {
    const tree = renderer.create(<Board />).toJSON();
    expect(tree.children[0]).toMatchSnapshot();
  });
});
