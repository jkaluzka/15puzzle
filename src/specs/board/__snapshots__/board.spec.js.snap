// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board renders Board component correctly 1`] = `
<h1>
  15 Puzzle Boards
</h1>
`;
